plyara/utils.py

```diff
@@ -251,14 +247,16 @@ def generate_logic_hash(rule):
 
 
 def rebuild_yara_rule(rule):
-    """Take a parsed yararule and rebuild it into a usable one.
+    """Take a parsed yararule and rebuild it into a usable one (DEPRECATED).
```

Comment:
why are we deprecating this?

Reply:
It's a "soft" deprecation. The way it works now it doesn't work on many rulesets for rebuilding. I'm open to fixing it too as another option.

Reply:
Also, the fixed util should have a different function name anyway. This name is taken and what output it has is now expected.